package.json

```diff
@@ -35,6 +35,8 @@
   },
   "main": "out/node/entry.js",
   "devDependencies": {
+    "@parcel/core": "^2.0.0-beta.3.1",
+    "@parcel/types": "^2.0.0-alpha.3",
```

Comment:
I'm not sure if we're even using the `types`? 🤔 At least since they're Flow and not TS, I don't know if we need them.